### PR TITLE
Add exercise

### PR DESCRIPTION
**Pull request by gonzaloserrano**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,14 +52,16 @@ In order to create a new space you need to perform a POST request to the `/space
...
 
-```
-    # Please, do not run this command from inside the repo. You want these tools to be downloaded as binaries not as dependencies for the project
-	go get -u github.com/golangci/golangci-lint/cmd/golangci-lint
-	go get -u github.com/mgechev/revive
-	go get -u github.com/matryer/moq
-```
+## TODO
+
+We expect from you to fork this project and design, implement and document the following use case: add a `Package` to a `Space`.
```
Comment (maitesin) [inline README.md:65]:
```suggestion
We expect from you to fork this project and design, implement and document the following use case: add a `Package` to an `Space`.
```

**Comment by maitesin** [comment]
Should we add here in the README what we are expecting from their solution. Regarding unit test, clean code, etc?

**Review by maitesin — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (toffentoffen) [inline README.md:65]: I would suggest to use the Domain word pairing to highlight the next phrase and make it more understandable.
```suggestion
We expect from you to fork this project and design, implement and document the following use case: **Pair a `Package` to a `Space`**.
```

Comment (toffentoffen) [inline README.md:65]: I would suggest to use the Domain word pairing to highlight the next phrase and make it more understandable.
```suggestion
We expect from you to fork this project and design, implement and document the following use case: **Pair a `Package` to a `Space`**.
```

**Review by toffentoffen — COMMENTED** [review]
All good. Just a small suggestion to make it easier and more comprehensive. I struggled a bit to connect the dots.

File: README.md [hunk 2]
```diff
@@ -47,14 +52,23 @@ In order to create a new space you need to perform a POST request to the `/space
...
+We expect from you to fork this project and design, implement and document the following use case: **Pair a `Package` to a `Space`**.
+Given a `Package` barcode and a `Space` barcode, pairing them means the Space contains the Package. 
+The operation should be idempotent. A new HTTP endpoint should be created. Existing endpoints should be compatible.
+
+We will evaluate:
+- the correctness and simplicity of the solution
+- testing
+- documentation
+
+Bonus: there is a suble bug in the in-memory implementations of the repositories. We will value if you reproduce the issue with a test and then fix it.
```
Comment (maitesin) [inline README.md:74]:
```suggestion
Bonus: there is a subtle bug in the in-memory implementations of the repositories. We will value if you reproduce the issue with a test and then fix it.
```

**Review by maitesin — APPROVED** [review]

**Pull request merged**